bin/tests/named_https_tests.rs

```diff
@@ -9,7 +9,7 @@
 #![cfg(feature = "dns-over-https")]
 
 #[macro_use]
-extern crate log;
+extern crate tracing;
```

Comment:
I think we can get rid of all these tracing and macro_use externs. They shouldn't be necessary anymore.